invoice/src/test/java/org/killbill/billing/invoice/usage/TestContiguousIntervalCapacityInArrear.java

```diff
@@ -243,6 +246,224 @@ public boolean apply(final InvoiceItem input) {
+            @Override
+            public boolean apply(final InvoiceItem input) {
+                return input.getAmount().compareTo(BigDecimal.ZERO) > 0;
+            }
+        }));
+
+
+        assertEquals(result.get(0).getAmount().compareTo(new BigDecimal("1.0")), 0, String.format("%s != 1.0", result.get(0).getAmount()));
+        assertEquals(result.get(0).getCurrency(), Currency.BTC);
+        assertEquals(result.get(0).getAccountId(), accountId);
+        assertEquals(result.get(0).getBundleId(), bundleId);
+        assertEquals(result.get(0).getSubscriptionId(), subscriptionId);
+        assertEquals(result.get(0).getPlanName(), planName);
+        assertEquals(result.get(0).getPhaseName(), phaseName);
+        assertEquals(result.get(0).getUsageName(), usage.getName());
+        assertTrue(result.get(0).getStartDate().compareTo(startDate) == 0);
+        assertTrue(result.get(0).getEndDate().compareTo(firstBCDDate) == 0);
+        // check item detail
+        List<UsageInArrearDetail> itemDetails1 = objectMapper.readValue(result.get(0).getItemDetails(), new TypeReference<List<UsageInArrearDetail>>() {});
+        assertEquals(itemDetails1.size(), 2);
+        assertEquals(itemDetails1.get(0).getTier(), 1);
+        assertEquals(itemDetails1.get(0).getQuantity().intValue(), 99);
+        assertEquals(itemDetails1.get(0).getTierPrice().compareTo(BigDecimal.ONE), 0);
+        assertEquals(itemDetails1.get(1).getTier(), 1);
+        assertEquals(itemDetails1.get(1).getQuantity().intValue(), 5);
+        assertEquals(itemDetails1.get(1).getTierPrice().compareTo(BigDecimal.ONE), 0);
+
+        assertEquals(result.get(1).getAmount().compareTo(new BigDecimal("10.0")), 0, String.format("%s != 10.0", result.get(0).getAmount()));
+        assertEquals(result.get(1).getCurrency(), Currency.BTC);
+        assertEquals(result.get(1).getAccountId(), accountId);
+        assertEquals(result.get(1).getBundleId(), bundleId);
+        assertEquals(result.get(1).getSubscriptionId(), subscriptionId);
+        assertEquals(result.get(1).getPlanName(), planName);
+        assertEquals(result.get(1).getPhaseName(), phaseName);
+        assertEquals(result.get(1).getUsageName(), usage.getName());
+        assertTrue(result.get(1).getStartDate().compareTo(firstBCDDate) == 0);
+        assertTrue(result.get(1).getEndDate().compareTo(secondBCDDate) == 0);
+        // check item detail
+        List<UsageInArrearDetail> itemDetails2 = objectMapper.readValue(result.get(1).getItemDetails(), new TypeReference<List<UsageInArrearDetail>>() {});
+        assertEquals(itemDetails2.size(), 2);
+        assertEquals(itemDetails2.get(0).getTier(), 2);
+        assertEquals(itemDetails2.get(0).getQuantity().intValue(), 101);
+        assertEquals(itemDetails2.get(0).getTierPrice().compareTo(BigDecimal.TEN), 0);
+        assertEquals(itemDetails2.get(1).getTier(), 2);
```

Comment:
I think this should be tier `1` -- as detailed per [here](https://github.com/killbill/killbill-integration-tests/pull/13/files#r166222783)